cores/esp32/HWCDC.cpp

```diff
@@ -407,4 +568,4 @@ HWCDC USBSerial;
 #endif
 #endif
 
-#endif /* CONFIG_TINYUSB_CDC_ENABLED */
+#endif /* CONFIG_IDF_TARGET_ESP32C3 */
```

Comment:
To be accurate :)
```suggestion
#endif /* CONFIG_IDF_TARGET_ESP32C3 || CONFIG_IDF_TARGET_ESP32S3 */
```

Reply:
I see no need for that. It is the same as in the .h file.